bookwyrm/templates/snippets/follow_button.html

```diff
@@ -12,6 +12,7 @@
 
 <div class="field{% if not minimal %} has-addons{% else %} mb-0{% endif %}">
     <div class="control">
+        {% if not followers_page %}
         <form action="{% url 'follow' %}" method="POST" class="interaction follow_{{ user.id }} {% if relationship.is_following or relationship.is_follow_pending %}is-hidden{%endif %}" data-id="follow_{{ user.id }}">
```

Comment:
Implemented like this, it is no longer possible to follow anybody from the followers page.

Since I think following/unfollowing are going to be more likely than removing follows... about keeping the current Follow/Unfollow button here, and have "Remove follow" move to the menu? (With an `is-danger` class, probably below _Block_.)

P.S.: I tried to create a commit that moves it but, unlike the `/block` endpoint, `/remove-follow` does not take any parameters, and I wasn't sure how to proceed.

Reply:
Oh yeah, somehow that hadn't occurred to me, oops!

> Since (I assume) following/unfollowing are going to be more common than removing follows... about keeping the current Follow/Unfollow button here, and have "Remove follow" move to the menu? (With an is-danger class, probably below Block.)

This is a good suggestion, I'll adjust.